Replicator/c4ReplicatorImpl.hh

```diff
@@ -475,7 +475,7 @@ namespace litecore {
                 if (status.progress.unitsTotal > 0)
                     progress = 100.0 * double(status.progress.unitsCompleted)
                                      / status.progress.unitsTotal;
-                if (status.error.code) {
+                if (status.error.code && status.level <= kC4Offline) {
```

Comment:
Not sure if this is requested by the ticket.

Reply:
It is not.  This merely prevents a log statement from coming.  What the ticket is after is stopping the entire callback from happening.

Reply:
To update this PR; Jianmin, Blake and I have discussed it. What is necessary to solve the ticket is to clear the error code from the status when level <= kC4Offline. Not something I understood before